giza_actions/agent.py

```diff
@@ -376,17 +378,29 @@ def _wait_for_proof(
             self._endpoint_id, self._proof_job.request_id
         )
 
-    def _verify_proof(self, client: EndpointsClient) -> bool:
```

Comment:
You are removing a function that its necessary, it seems that something has happened while updating your branch

Reply:
Since is just this file, im gonna add it those functions by myself and see if it works.